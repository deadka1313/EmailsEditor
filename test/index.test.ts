import CreateEmailForm from '../src';

describe('Test CreateEmailForm', () => {
    const div = document.createElement('div');
    const createEmailForm = new CreateEmailForm(div);

    test('test setEmail', () => {
        expect(createEmailForm.setEmail()).toBe('setEmail');
    });
    test('test getEmail', () => {
        expect(createEmailForm.getEmail().length).toBe(3);
    });
});
