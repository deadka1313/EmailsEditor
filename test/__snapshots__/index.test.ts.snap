// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test DOM test snapshot 1`] = `<div />`;
